ambry-api/src/main/java/com/github/ambry/config/FrontendConfig.java

```diff
@@ -314,7 +314,7 @@ public FrontendConfig(VerifiableProperties verifiableProperties) {
     maxAcceptableTtlSecsIfTtlRequired = verifiableProperties.getIntInRange(MAX_ACCEPTABLE_TTL_SECS_IF_TTL_REQUIRED_KEY,
         (int) TimeUnit.DAYS.toSeconds(30), 0, Integer.MAX_VALUE);
     maxJsonRequestSizeBytes =
-        verifiableProperties.getIntInRange(MAX_JSON_REQUEST_SIZE_BYTES_KEY, 20 * 1024 * 1024, 0, Integer.MAX_VALUE);
+        verifiableProperties.getIntInRange(MAX_JSON_REQUEST_SIZE_BYTES_KEY, 128 * 1024 * 1024, 0, Integer.MAX_VALUE);
```

Comment:
do we need 128MB to be the default value? or can we just change the value for this configuration in some cluster?

Reply:
good idea. yes we can change the config for a specific cluster.